default-templates/primary-ignition.tmpl

```diff
@@ -0,0 +1,56 @@
+ignition_version: 1
+systemd:
+  units:
+    - name: update-engine.service
+      enable: false
+      mask: true
+    - name: locksmithd.service
+      enable: false
+      mask: true
+    - name: etcd2.service
+      enable: true
+      dropins:
+       - name: 30-etcd2-kocho.conf
+         contents: |
+           [Unit]
+           Requires=coreos-metadata.service
+           After=coreos-metadata.service
```

Comment:
Variable substitution doesn't work with ignition like was working for cloudconfig. To get the value of this environment variables we need to make use of coreos-metadata service as done in here.